cmd/aws-vpc-cni-init/main.go

```diff
@@ -110,13 +113,16 @@ func configureIPv6Settings(procSys procsyswrapper.ProcSys, primaryIF string) err
 		}
 		val, _ := procSys.Get(entry)
 		log.Infof("Updated %s to %s", entry, val)
-
-		entry = "net/ipv6/conf/all/forwarding"
+	}
+	// Check if IPv6 egress supporting is enabled in IPv4 cluster
+	ipv6EgressEnabled := utils.GetBoolAsStringEnvVar(envEgress, defaultEnableIPv6Egress)
+	if enableIPv6 || ipv6EgressEnabled {
+		entry := "net/ipv6/conf/all/forwarding"
 		err = procSys.Set(entry, "1")
 		if err != nil {
 			return errors.Wrap(err, "Failed to enable ipv6 forwarding")
```

Comment:
nit: inconsistent casing of ipv6 vs IPv6 (https://github.com/aws/amazon-vpc-cni-k8s/blob/3d9e67decb355cf39cf5cbc5927484d9ecc8dda9/cmd/aws-vpc-cni-init/main.go#L175) in error log messages

Reply:
Thanks, will use `IPv6` in logs for consistency.